src/lib/services/environment-service.ts

```diff
@@ -52,10 +57,20 @@ export default class EnvironmentService {
         this.flagResolver = flagResolver;
     }
 
-    async getAll(): Promise<IEnvironment[]> {
+    async getAll(): Promise<IEnvironmentWithCounts[]> {
         return this.environmentStore.getAllWithCounts();
     }
 
+    async getWithCounts(name: string): Promise<IEnvironmentWithCounts> {
+        let matches = await this.environmentStore.getAllWithCounts({
+            name: name,
```

Comment:
More concise way to do it
```suggestion
            name,
```